index.js

```diff
@@ -249,6 +250,48 @@ SteamCommunity.prototype.oAuthLogin = function(steamguard, token, callback) {
 	}, "steamcommunity");
 };
 
+SteamCommunity.prototype.openidLogin = function(url, callback) {
+    var self = this;
+    self.loggedIn(function(err, loggedIn) {
```

Comment:
Perhaps this check could be removed. Let the user handle this on their own.

Reply:
For example, the snippet in the first comment makes an unnecessary request since we know for sure we are logged in.

Reply:
You got a point. I just wanted it to be robust, but performance is a good quality too.